packages/cli/src/utils/object-to-error.ts

```diff
@@ -2,6 +2,8 @@ import { isObjectLiteral } from 'n8n-core';
 import { NodeOperationError } from 'n8n-workflow';
 import type { Workflow } from 'n8n-workflow';
 
+const transientErrorProperties = ['description', 'stack', 'executionId', 'workflowId'];
```

Comment:
how about just `errorProperties`, with a jsdoc defining what these are for. 
I prefer using doc-lines over variable names to define what the variable is for.